pages/transactional-email/how-to/authenticate-domain.mdx

```diff
@@ -0,0 +1,128 @@
+3. Enter your desired tags and their values in the **Value** section. Make sure that you add the mandatory `v` and `p` tags, and that you separate tags from each other with semi-colons. Find out about DMARC configuration in our [dedicated documentation](/transactional-email/reference-content/dmarc-configuration/).
+    Your **Value** field should look similar to the following
+    ```
+    v=DMARC1; p=none; rua=mailto:your@email.com
+    ```
+    <Message type="important">
+     It is possible to configure several DMARC records for one domain, but they must have a different name to be valid.
+     For example, the `_dmarc.mydomain.com` and `_dmarc.tem.mydomain.com` DMARC records are valid.
+    </Message>
+6. Confirm the changes and add the DMARC record.
+
+    <Message type="tip">
+     If you have a domain at Scaleway Domains and DNS, you can access your account from the Scaleway [console](https://console.scaleway.com/), click **Domains and DNS** > **Your domain** > **DNS Zones**, then add a record in the same zone as your Transactional Email's domain.
+    </Message>
+
+### How to configure Scaleway's DMARC record
+
+1. Access your domain's **DNS Verification** tab and scroll down to the **DMARC record** section.
+2. Copy the values that display in the **Name** and the **Value** fields.
+3. Add a `TXT` record type in the same zone as your Transactional Email's domain, on your DNS zone.
+4. In the **Name** field, paste the value you have copied in step 2, followed by the domain name.
+5. In the **Value** field, paste the value you have copied in step 2. Your **Value** field should look like the following:
+    ```
+    v=DMARC1; p=none;
+    ```
+6. Confirm the changes and add the DMARC record.
+
+## How to configure MX records
+
+<Message type="important">
+  We recommend you use your own MX server if you have one. If you do not have an MX server, we recommend you use Scaleway's blackhole MX, as it allows you to avoid email rejection. However, this also means that the emails sent back to the Scaleway blackhole MX, will be lost and unrecoverable.
+</Message>
+
+1. Add an [MX record](/transactional-email/concepts/#mx-record) type in the root zone of your domain on your DNS zone.
+2. Enter a name of your choice in the **Name** field.
+3. Set `10` as a [priority](/transactional-email/concepts/#priority) for your MX record.
+    <Message type="note">
+     Priority indicates the order in which your mail servers should be tried by a mail server that is attempting to deliver emails.
+    </Message>
+4. Enter the name of your MX followed by a trailing dot in the **Hostname** field.
+5. Save your changes.
+
+### How to add Scaleway's MX record
+
+<Message type="important">
+  - Scaleway provides its own MX server to improve email deliverability. <br /> However, Scaleway's MX server is a **blackhole**. This means that **any email sent back to it, will be lost with no possibility of recovery**.
```

Comment:
```suggestion
  - Scaleway provides its own MX server to improve email deliverability. <br /> However, Scaleway's MX server is a **blackhole**. This means that **any email sent back to it will be lost with no possibility of recovery**.
```